services/app-web/src/components/SubmissionSummarySection/RateDetailsSummarySection/SingleRateSummarySection.test.tsx

```diff
@@ -525,4 +445,212 @@ describe('SingleRateSummarySection', () => {
             ).toBeNull()
         })
     })
+
+    describe('Missing data error notifications', () => {
```

Comment:
Just curious why did this summary section get its own tests but the others did not and were just tested view the `ReviewSubmit.test.tsx`

Reply:
I'm glad you asked. I think this could be changed at a later time to follow the pattern, but it's this way because it's only used in `RateSummary.tsx` which is hardcoded to pass in true for prop `isSubmitted` in `SingleRateSummary`.

Usuage in `RateSummary.tsx`
```ts
...
<SingleRateSummarySection
    rate={rate}
    isSubmitted // can assume isSubmitted because we redirect for unlocked
    statePrograms={rate.state.programs}
/>
...
```

From the comment, it didn't make sense to me to test from the `RateSummary.test.ts` because:
1. I don't want to add complexity to `RateSummary` just so I can toggle isSubmitted only when testing.
2. The comment insinuates that this page should never show notifications anyways.
3. There is only 1 Summary component here that has all the data.
4. `SingleRateSummarySection` seems to be the best place to test this even though there does not seem to be a case where this component would show notifications, but because it has the functionality we should have tests to incase this functionality is used.